examples/assets/u2f-api.js

```diff
@@ -0,0 +1,651 @@
+// Copyright 2014-2015 Google Inc. All rights reserved.
```

Comment:
Personally I'm not too happy with embedding a client library in a server component. But considering that it is within `examples/` this may be acceptable.

Alternatively _IF_ the file on demo.yubico.com never changes content (Subresource Integrity) nor file path then we could embed it as well.